pyfarm/jobtypes/core/jobtype.py

```diff
@@ -890,6 +890,96 @@ def set_task_progress(self, task, progress):
                     "%r." % (task["id"], error))
                 raise
 
+    @inlineCallbacks
+    def set_task_started_now(self, task):
```

Comment:
I was hoping to leave picking any actual timestamps to the master (so we have a consistent time source). Also, I don't really see a use-case for setting any other times.